subsys/bluetooth/host/l2cap_internal.h

```diff
@@ -360,3 +360,12 @@ void l2cap_br_encrypt_change(struct bt_conn *conn, uint8_t hci_status);
 
 /* Handle received data */
 void bt_l2cap_br_recv(struct bt_conn *conn, struct net_buf *buf);
+
+struct bt_l2cap_cb {
+	void (*ecred_channels_connect_rsp)(struct bt_conn *conn, uint16_t result, uint8_t attempted,
```

Comment:
Please use some shorter identifiers, eg `ecred_conn_rsp` (I assume “chan” is implied)